### PR TITLE
Add 3.5.0 changelog

### PR DESCRIPTION
**Pull request by akofink**

### CODE REVIEW

**Comment by ehelms** [comment]
Going to wait on merging this as this can change based on whats included in RC1.

**Comment by akofink** [comment]
Updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+ * Update installation media paths in k-change-hostname ([#20987](http://projects.theforeman.org/issues/20987), [f1418fc0](http://github.com/katello/katello-packaging/commit/f1418fc08a9f2614b0f136bde6469328f0b6bf34))
+ * katello-change-hostname should print command output when it errors out. ([#20984](http://projects.theforeman.org/issues/20984), [a727125b](http://github.com/katello/katello-packaging/commit/a727125bb492abdf14c9eb10c39e8d67d9864cf9))
+ * k-change-hostname can mix up internal capsule names ([#20983](http://projects.theforeman.org/issues/20983), [b1a9d445](http://github.com/katello/katello-packaging/commit/b1a9d44581e7e93131c8cadfe04cb0b2d32b2346))
+ * clean_installed_packages script logs a count(*), causing slow performance ([#20946](http://projects.theforeman.org/issues/20946), [cc0b15fb](http://github.com/katello/katello/commit/cc0b15fb330178aca8317873eaaf2ef1b280d1d7))
+ * k-change-hostname will check for exit code on a skipped command on a foreman-proxy ([#20944](http://projects.theforeman.org/issues/20944), [d32d5854](http://github.com/katello/katello-packaging/commit/d32d5854c167efa0bf2dc02fa5ceceb07f2eff71))
+# 3.5.0 Schwarzbier (2017-09-21)
```
Comment (ehelms) [inline CHANGELOG.md]: Duplicated here -- seems like a bug in tool_belt

Comment (akofink) [reply CHANGELOG.md]: Ah! No bug. I just realized that it amends the file, not overwrites it, so you have to remove the changelog when you regenerate it.

**Pull request merged**